models/models.js

```diff
@@ -49,10 +49,15 @@ exports.insertCommentByReviewId = (review_id, { username, body }) => {
 };
 
 exports.updateVotesByReviewId = (inc_votes, review_id) => {
-    return utilCheckPositiveVotes(review_id, inc_votes)
+    return utilCheckReviewExist(review_id)
         .then(() => {
-            return db.query('UPDATE reviews SET votes = votes + $1 WHERE review_id = $2 AND votes + $1 >= 0 RETURNING *;', [inc_votes, review_id]);
+            return db.query('UPDATE reviews SET votes = votes + $1 WHERE review_id = $2 RETURNING *;', [inc_votes, review_id]);
         }).then(updatedVote => {
+            if (updatedVote.rows[0].votes < 0) {
+                const newObj = Object.assign({}, updatedVote.rows[0]);
+                newObj.votes = 0;
+                return newObj;
+            }
```

Comment:
Be careful here.  You are checking if the object returned from the query has negative votes, and then sending back a new object where votes = 0, but the value in the database will still be negative.

There are a couple of things you can do to fix it (up to you):
-allow negative votes
-another query inside the if statement that 'if votes is negative, set votes to 0'
-change your initial query so that votes doesn't go below zero (trickiest)